### PR TITLE
[ISSUE-3083][Improve] Improve streampark-flink module base on [3.2 Constant Variables Definition]

### LINKED ISSUE #3083: [Improve] Improve `streampark-flink` module base on [3.2 Constant Variables Definition]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

 [Improve] Improve `streampark-flink` module base on [3.2 Constant Variables Definition](https://streampark.apache.org/community/submit_guide/code_style_and_quality_guide#32-constant-variables-definition)

### Usage Scenario

_No response_

### Related issues

#2986 

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by RocMarshal**
## What changes were proposed in this pull request

Issue Number: close #3083 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

File: streampark-common/src/main/scala/org/apache/streampark/common/util/YarnUtils.scala [hunk 1]
```diff
@@ -127,8 +128,8 @@ object YarnUtils extends Logger {
         val conf = HadoopUtils.hadoopConf
         val useHttps = YarnConfiguration.useHttps(conf)
         val (addressPrefix, defaultPort, protocol) = useHttps match {
-          case x if x => (YarnConfiguration.RM_WEBAPP_HTTPS_ADDRESS, "8090", "https://")
-          case _ => (YarnConfiguration.RM_WEBAPP_ADDRESS, "8088", "http://")
+          case x if x => (YarnConfiguration.RM_WEBAPP_HTTPS_ADDRESS, "8090", Constant.HTTPS_SCHEMA)
+          case _ => (YarnConfiguration.RM_WEBAPP_ADDRESS, "8088", Constant.HTTP_SCHEMA)
```
Comment (wuchunfu) [inline streampark-common/src/main/scala/org/apache/streampark/common/util/YarnUtils.scala:132]: Hi, @RocMarshal  it is recommended that the port number here be extracted as a constant.

Comment (RocMarshal) [reply streampark-common/src/main/scala/org/apache/streampark/common/util/YarnUtils.scala:132]: Hi, @wuchunfu Thanks for the review .

image

I rechecked the rule. 
I'd appreciated with your confirmation~

Comment (wuchunfu) [reply streampark-common/src/main/scala/org/apache/streampark/common/util/YarnUtils.scala:132]: @RocMarshal Thank you for reminding me. I remembered it wrong. I agree with you.

**Review by wuchunfu — APPROVED** [review]
LGTM

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**